content-resources/src/main/java/flyway/announcements/V1_0_0__register_announcement_bundles.java

```diff
@@ -0,0 +1,25 @@
+package flyway.announcements;
+
+import java.sql.Connection;
+import org.flywaydb.core.api.migration.BaseJavaMigration;
+import org.flywaydb.core.api.migration.Context;
+
+import org.oskari.helpers.BundleHelper;
+import fi.nls.oskari.domain.map.view.Bundle;
+
+public class V1_0_0__register_announcement_bundles extends BaseJavaMigration {
+
+    @Override
+	public void migrate(Context context) throws Exception {
+        Connection connection = context.getConnection();
+		
+		// BundleHelper checks if these bundles are already registered
+        Bundle bundle = new Bundle();
+        bundle.setName("announcements");
+        BundleHelper.registerBundle(connection, bundle);
+
+        Bundle admin = new Bundle();
+        admin.setName("admin-announcements");
+        BundleHelper.registerBundle(connection, admin);
```

Comment:
```suggestion
        // BundleHelper checks if these bundles are already registered
        BundleHelper.registerBundle(connection, "announcements");
        BundleHelper.registerBundle(connection, "admin-announcements");
```

Reply:
This migration should be moved to the oskari/core-module instead of having a new flyway module just for bundle registration.

Reply:
What file/folder exactly?

Reply:
The same you have for the db table: `flyway/oskari` -> `package flyway.oskari`. The flyway modules are grouped by the folder they are in so with this we would have an `announcements` module with a single migration registering the bundles. If we would want to separate announcements functionality to it's own module the table creation should be in this module as well. But I think it's easier for everyone if we just have both of them in the core-module that can't be skipped.